src/openrct2/network/Socket.cpp

```diff
@@ -22,6 +22,8 @@
 #include <cmath>
 
 #ifdef _WIN32
+    #pragma comment(lib, "Ws2_32.lib")
```

Comment:
What is this here for?

Reply:
This links the winsock library, used to be in Network.cpp rather than Socket